connector/spanmetricsconnector/README.md

```diff
@@ -33,6 +33,29 @@ across all spans:
 - `span.kind`
 - `status.code`
 
+
+## Span to Metrics processor to Span to metrics connector
+
+The spanmetrics connector is a port of the [spanmetrics](../../processor/spanmetricsprocessor/README.md) processor, but with multiple improvements
+and breaking changes. It was done to bring the `spanmetrics` connector closer to the OpenTelemetry
+specification and make the component agnostic to exporters logic. The `spanmetrics` processor
+essentially was mixing the OTel with Prometheus conventions by using the OTel data model and
+the Prometheus metric and attributes naming convention.
+
+The following changes were done to the connector component.
+
+Breaking changes:
+- The `operation` metric attribute was renamed to `span.name`.
+- The `latency` histogram metric name was changed to `duration`.
+- The `_total` metric prefix was dropped from generated metrics names.
+- The Prometheus-specific metrics labels sanitization was dropped.
+
+Improvements:
+- Added support for OTel exponential histograms for recording span duration measurements.
+- Added support for the milliseconds and seconds histogram units.
+- Added support for generating metrics resource scope attributes. Note that this change might result
```

Comment:
This would generate more metrics for _any_ exporter right, not just Prometheus?

Further, I think _might_ is too weak. If a user has 10 pods, they'll generate 10x as many metrics. Could we have a stronger call out here?

Reply:
> This would generate more metrics for any exporter right, not just Prometheus?

I cannot talk about all exporters and their behavior. 

>  If a user has 10 pods, they'll generate 10x as many metrics. 

yes, there will be 10x resource scope one per entity that generates telemetry data.